### PR TITLE
Fix/ticket reassign issue

### PR DESCRIPTION
**Pull request by xi2677-saimounikaduvvuru**
Dropdown list in the ticket listing page will now show all the admin users from all the departments, whereas the dropdown in the employee dashboard page will show only those admin users from the current user's department.

### CODE REVIEW

File: src/core/EmployeeDasboard/EmployeeDashboardBody.js [hunk 1]
```diff
@@ -43,14 +43,15 @@ const EmployeeDashboardBody = (props) => {
   };
   useEffect(() => {
     dispatch(resetData());
+    console.log(commonState.allAdminData.allAdminUsersInDept.length);
```
Comment (sRajat98) [inline src/core/EmployeeDasboard/EmployeeDashboardBody.js]: remove console.log

File: src/app/redux/reducers/commonReducers.js [hunk 2]
```diff
@@ -72,6 +75,25 @@ const commonReducer = (state = initalState, action) => {
           allAdminUsersFailure: true,
         },
       };
+    case types.GET_ALL_ADMIN_USERS_IN_DEPT_SUCCESS:
```
Comment (sRajat98) [inline src/app/redux/reducers/commonReducers.js:87]: i see only success and failure scenarios. Please add a loading condition as well

File: src/app/redux/actions/commonActions.js [hunk 3]
```diff
@@ -37,12 +37,30 @@ export const getAllAdminsInDept = (departmentId) => (dispatch) => {
   );
 };
```
Comment (sRajat98) [inline src/app/redux/actions/commonActions.js:39]: Add a loading action

**Review by sRajat98 — CHANGES_REQUESTED** [review]

File: src/core/EmployeeDasboard/EmployeeDashboardBody.js (hunk 1 above)
Comment (xi2677-saimounikaduvvuru) [reply src/core/EmployeeDasboard/EmployeeDashboardBody.js]: Removed

File: src/app/redux/actions/commonActions.js (hunk 3 above)
Comment (xi2677-saimounikaduvvuru) [reply src/app/redux/actions/commonActions.js:39]: Added

File: src/app/redux/reducers/commonReducers.js (hunk 2 above)
Comment (xi2677-saimounikaduvvuru) [reply src/app/redux/reducers/commonReducers.js:87]: Added loading condition

**Review by sRajat98 — APPROVED** [review]

**Pull request merged**